interfaces/nginx_route/v0/schema.py

```diff
@@ -0,0 +1,149 @@
+from __future__ import annotations
+
+from typing import Optional
+
+from pydantic import BaseModel, Field
+
+
+class NginxRouteRequirerSchema(BaseModel):
+    service_hostname: str = Field(
+        alias="service-hostname",
+        description="The hostname of the service to create an ingress for.",
+        examples=["example.com"],
+        title="Service Hostname",
+    )
+    service_name: str = Field(
+        alias="service-name",
+        description="The name of the service to create an ingress for.",
+        examples=[],
```

Comment:
Please provide examples (applicable elsewhere in this file as well)

Reply:
Okay, I added two examples `"wordpress"` and `"indico"`, thanks.

Reply:
There's still examples missing for `max_body_size` and other fields, I think it's useful to provide examples in all the fields.

Reply:
The max_body_size is an integer value set arbitrarily. The remaining fields are either integer or boolean values, except for the tis-secret-name, which can be any string that represents a Kubernetes secret name.

I have provided an example for the tls-secret-name. Would you like me to provide some examples of random integer and boolean values for the integer and boolean fields?

Reply:
Ok thanks